src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityDetail/detail.html.twig

```diff
@@ -5,6 +5,7 @@
 
     <h1>{% block page_heading %}{{ 'entity.detail.title'|trans }}{% endblock %}</h1>
 
+    <h2>{{ isAdmin }}</h2>
```

Comment:
I think this was carried over from an old branch? This `isAdmin` h2 was removed in another dev-effort IIRC

Reply:
/me glares at git